websites/factories.py

```diff
@@ -16,39 +16,37 @@
 """
 
 
+class WebsiteStarterFactory(DjangoModelFactory):
+    """Factory for WebsiteStarter"""
+
+    path = factory.Faker("uri")
+    name = factory.Faker("domain_word")
+    slug = factory.Sequence(lambda n: "starter-%x" % n)
+    source = FuzzyChoice(STARTER_SOURCES)
+    commit = factory.Faker("md5")
+    config = EXAMPLE_SITE_CONFIG
+
+    class Meta:
+        model = WebsiteStarter
+
+
 class WebsiteFactory(DjangoModelFactory):
     """Factory for Website"""
 
-    title = factory.Sequence(lambda n: "OCW Course %s" % n)
-    url_path = factory.Sequence(lambda n: "/ocw_site_x/%s" % n)
-    type = FuzzyChoice((WEBSITE_TYPE_COURSE, "other"))
-
+    title = factory.Sequence(lambda n: "Site %s" % n)
+    name = factory.Sequence(lambda n: "/site_x/%s" % n)
```

Comment:
Just curious, what does the `x` mean in `site_x`?

Reply:
No meaning. I'll fix